src/main/kotlin/org/malachite/estella/aplication/api/ApplicationController.kt

```diff
@@ -66,13 +69,24 @@ class ApplicationController(
 
     @CrossOrigin
     @GetMapping("/job-seeker")
-    fun getAllApplicationsByJobSeeker(): ResponseEntity<Any> =
+    fun getAllApplicationsByJobSeeker(): ResponseEntity<List<ApplicationDTOWithStagesListAndOfferName>> =
         securityService.getJobSeekerFromContext()
             ?.id
             ?.let { applicationService.getApplicationsByJobSeeker(it) }
-            ?.map { it.toApplicationDTO() }
+            ?.map { it.toDto() }
```

Comment:
Nie wiem czy bym nie zmienił tej nazwy bo mamy tyle tych DTO że nie wiele to mówi XDDD

Reply:
Wiem, ale popatrz sobie na definicję tej funkcji (kilka linijek niżej). Jak będę chciał napisać pełną nazwą, to przecież umrę XDDD

Reply:
Nawet skrótem jak walniesz `toApplicationDtoWSLAON` to imo warto